pkg/asset/installconfig/libvirt/libvirt.go

```diff
@@ -36,3 +38,14 @@ func Platform() (*libvirt.Platform, error) {
 func uriValidator(ans interface{}) error {
 	return validate.URI(ans.(string))
 }
+
+// AddWildcardDNS configures the host's dnsmasq, so that all apps can
+// found by Ingress Operator
+func AddWildcardDNS(domain string) error {
+	cmd := exec.Command("hack/configure-dnsmasq.sh", domain)
```

Comment:
Considering that the installer is a binary, is this portable or does this expect hack/configure-dnsmasq.sh to be in the $PATH?